Doc/library/string.rst

```diff
@@ -789,10 +792,20 @@ Nesting arguments and more complex examples::
 
 
 
-.. _template-strings:
+.. _template-strings-pep292:
```

Comment:
This will break anchor references within the page: https://docs.python.org/3/library/string.html#template-strings

Are we OK with doing so?